src/cmd/logs.rs

```diff
@@ -0,0 +1,78 @@
+//! The `logs` command.
+
+use args;
+use cmd::CommandCompose;
+use command_runner::CommandRunner;
+#[cfg(test)]
+use command_runner::TestCommandRunner;
+use errors::*;
+use pod::{Pod, PodType};
+use project::Project;
+
+/// We implement `logs` with a trait so we put it in its own module.
+pub trait CommandLogs {
+    /// Display logs for a given service
+    fn logs<CR>(&self,
+                runner: &CR,
+                act_on: &args::ActOn,
+                opts: &args::opts::Logs)
+                -> Result<()>
+        where CR: CommandRunner;
+}
+
+impl CommandLogs for Project {
+    fn logs<CR>(&self,
+                runner: &CR,
+                act_on: &args::ActOn,
+                opts: &args::opts::Logs)
+                -> Result<()>
+        where CR: CommandRunner
+    {
+        match *act_on {
+            args::ActOn::Named(ref names) if names.len() == 1 => {
+                let pred = |p: &Pod| p.pod_type() != PodType::Task;
```

Comment:
I figured all tasks would be run interactively but there's no _real_ reason I guess